src/Workspaces/Core/Portable/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class CodeStyleHelpers
+    {
+        public static CodeStyleOption<bool> ParseEditorConfigCodeStyleOption(string arg)
+        {
+            var args = arg.Split(':');
+            bool isEnabled = false;
+            if (args.Length != 2)
+            {
+                if (arg.Length == 1)
+                {
+                    if (bool.TryParse(args[0].Trim(), out isEnabled) && !isEnabled)
+                    {
+                        return new CodeStyleOption<bool>(value: isEnabled, notification: NotificationOption.None);
+                    }
+                    else
+                    {
+                        return CodeStyleOption<bool>.Default;
+                    }
+                }
+                return CodeStyleOption<bool>.Default;
```

Comment:
Is this right? Wouldn't we want to say that we should return the default for the option which might be different than this?